extras/linked-token/script/CallBurnAndTransfer.s.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import "forge-std/Script.sol";
+import "../src/LinkedTokenReplica.sol";
+
+contract CallBurnAndTransfer is Script {
+    function run(address contractAddress, address recipient, uint256 amount) public {
+        LinkedTokenReplica replica = LinkedTokenReplica(contractAddress);
+        vm.startBroadcast();
+        replica.linkedTransfer(recipient, amount);
+        vm.stopBroadcast();
+        // Log the address of the deployed contract
+        console.log("done");
+    }
+}
```

Comment:
Can we turn this into a cast command in the Makefile? (was review comment before already)

Reply:
817e6e75 pushed